java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestItemUtils.java

```diff
@@ -22,47 +22,53 @@
 import org.eclipse.jdt.core.ISourceReference;
 import org.eclipse.jdt.core.IType;
 import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.core.SourceRange;
 import org.eclipse.jdt.ls.core.internal.JDTUtils;
+import org.eclipse.jdt.ls.core.internal.hover.JavaElementLabels;
 import org.eclipse.lsp4j.Range;
 
 @SuppressWarnings("restriction")
 public class TestItemUtils {
 
     public static final String DEFAULT_PACKAGE_NAME = "<Default Package>";
 
-    public static TestItem constructTestItem(IJavaElement element, TestLevel level) throws JavaModelException {
-        return constructTestItem(element, level, null);
-    }
-
-    public static TestItem constructTestItem(IJavaElement element, TestLevel level, TestKind kind)
+    public static JavaTestItem constructJavaTestItem(IJavaElement element, TestLevel level, TestKind kind)
             throws JavaModelException {
         final String displayName;
         final String fullName;
         if (element instanceof IPackageFragment && ((IPackageFragment) element).isDefaultPackage()) {
             displayName = DEFAULT_PACKAGE_NAME;
-            fullName = DEFAULT_PACKAGE_NAME;
+            fullName = "";
```

Comment:
is empty string working?

Reply:
Yes default package's element name is empty.

While here I should get the fullName in just one call, no need to handle separately, will update it.